strings/numbers.cc

```diff
@@ -394,6 +394,39 @@ bool safe_strto64_base(StringPiece str,
   return safe_int_internal<int64>(str.begin(), str.end(), base, v);
 }
 
+bool safe_strtou32_base(StringPiece str,
```

Comment:
regarding `safe` functions - you have them via absl in numbers.h.
you can add u64tostr since I do not see it in absl besides https://github.com/abseil/abseil-cpp/blob/12bc53e0318d80569270a5b26ccbc62b52022b89/absl/strings/internal/numbers_test_common.h#L33

I prefer that you add a unit test to check against typos if you decide to add `u64tostr`

Reply:
These functions are only available in numbers.h in the internal namespace. Is it ok to use that namespace directly in code?